README.md

```diff
@@ -392,22 +392,41 @@ your-secretRootKey-value:
 
-During the transition period, there are two (or more) `ServiceBinding`: the original and the rotated one (holds the `services.cloud.sap.com/stale` label), which is deleted once the `rotatedBindingTTL` duration elapses.</br></br>
-You can also choose the `services.cloud.sap.com/forceRotate` annotation (value doesn't matter), upon which immediate credentials rotation is performed. Note that the prerequisite for the force action is that credentials rotation `enabled` field is set to true.).
+| Parameter         | Type     | Description                                                                                                                                               | Valid Values                                                                                                                                            |
+|:-----------------|:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled` | bool | Whether the binding rotation option is enabled.                                                                                                           |                                                                                                                                                         |
+| `rotationFrequency` | string | Indicates the interval between credential rotations. The actual frequency may be longer than the one you specified.  See details under **Notes** below. | "m", "h"                                                                                                                                                |                                                                                                                                                                |
+| `rotatedBindingTTL`   |  string | Specifies for how long to keep the old `ServiceBinding`. The actual TTL may be longer than the one you specified.  See details under **Notes** below.   | "m", "h"                                                                                                                                                |   
 
-**Note:**<br> It isn't possible to enable automatic credentials rotation to an already-rotated `ServiceBinding` (with the `services.cloud.sap.com/stale` label).
+### Example
+In the following example, the `ServiceBinding` resource is configured to rotate credentials every 25 days and to keep the old `ServiceBinding` for 2 days before deleting it.
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  credentialsRotationPolicy:
+    enabled: true
+    rotatedBindingTTL: 48h
+    rotationFrequency: 600h
+ ```
+
+**Notes**
```

Comment:
it shall not be note*s*, as this is a collection of notes that re not necessarily related.
have to be split.